### ISSUE TITLE
Call for maintainers

### ISSUE THREAD

**Issue by amacneil** [amacneil/dbmate#258]
Hi everyone,

As you can see, I haven't had a lot of time to work on dbmate lately. It works great for many use cases and doesn't need a lot of ongoing development, but as you can see there are occasional questions or feature requests that come up in GitHub.

I'm looking for people to help with issues and/or coding.

- For issues, we need help responding to questions, asking for more information, closing issues that aren't relevant, etc.
- For coding, we're not looking to make major changes. Dbmate has a "keep it simple" philosophy, and is largely feature complete. That said, if small features can be introduced without breaking existing users or making big refactors, then we're open to it. PRs need to be reviewed and either merged or closed with appropriate reasoning. I'm happy to continue to provide high level guidance on what is/isn't appropriate from a feature perspective, but I don't have time to review and merge PRs.

**Comment by Enrico204** [comment]
I might try to help with some issues. I'm using `dbmate` on daily basis in my lab with students :-)
I don't know how much time I will have in next months, but I'll try :-D

**Comment by rohitpaulk** [comment]
Would love to help - I've been using dbmate in a few projects, and recently published a buildpack for using it on Heroku: https://github.com/codecrafters-io/heroku-buildpack-dbmate. 

Added notifications for all activity on this repository - will keep an eye out!

**Comment by amacneil** [comment]
@rohitpaulk added!

**Comment by paulovitorbal** [comment]
I known too little about GO lang, but I will try to help, as long you can review my PRs. I can tackle some small issues/features. What do you think?

**Comment by docapotamus** [comment]
Hi @amacneil. We're making use of dbmate at work. I would love to help out. There a few features we would like to at add in the coming months so more than happy to help.

**Comment by imsky** [comment]
@amacneil happy to help 👋

**Comment by amacneil** [comment]
Great!

**Comment by Mizaro** [comment]
@amacneil I would like to help as well 👋🏻

**Comment by dossy** [comment]
It looks like there are plenty of volunteers, but in just in case more are needed, I can help out as well.  👋

**Comment by gregwebs** [comment]
@amacneil I left comments on issues and PRs. It looks like there are several PRs open that would be backwards compatible and straightforward to merge. I think we could ask those on this thread to try out a beta release.

**Comment by gregwebs** [comment]
To help with maintenance I would suggest trying to find a maintainer for each individual driver and trying to hand over the reigns for the driver code as quickly as possible. It's quite difficult to maintain a driver that you aren't actively using.

**Comment by gregwebs** [comment]
@amacneil do you want to make @FatherCandle the maintainer of the ClickHouse driver?

**Comment by Mizaro** [comment]
@gregwebs I agree with your idea. I do think we should have some sort of coordination to avoid breaching to much.

I am using ClickHouse driver as well

**Comment by gregwebs** [comment]
This project could be re-structured so that there would be a new repo dbmate-clickhouse. That way the maintainer can have full rights there. The repos could exist under a dbmate organization.

**Comment by Mizaro** [comment]
@gregwebs Sounds excellent to me. How do you think we should proceed with it?

**Comment by gregwebs** [comment]
Calling a fork of the repo dbmate-clickhouse and developing the clickhouse driver there seems like a pragmatic approach right now.
Publish builds there called dbmate-clickhouse.

When the dbmate maintainers are ready they can either pull the latest clickhouse driver code in or decide to delete the code from dbmate and direct clickhouse users to dbmate-clickhouse. They could also decide to move much of main.go under `pkg/` so that it can be imported as a go package.

**Comment by amacneil** [comment]
Separate repos sounds overly complicated imo (any changes to core will require potential changes in all the driver repos).

If we're all sensible adults, people can maintain drivers in this repo. Let's try to get community consensus on any larger changes tho. 

@gregwebs @FatherCandle I gave you guys commit access, use it wisely :)

**Comment by lightSoulDev** [comment]
@amacneil, 👋🏻 I'm willing to help. I actively use dbmate in all my Go projects related to PostgreSQL. I have some features in mind that would benefit many devs, particularly in the realm of integration tests.

**Comment by melroy89** [comment]
Its now almost 2026.. is this still an issue or?
